webapps-schema/src/main/resources/schema/elasticsearch/create/index/operate-process.json

```diff
@@ -48,6 +48,12 @@
       "formId": {
         "type": "keyword"
       },
+      "formKey": {
+        "type": "keyword"
+      },
+      "isFormEmbedded": {
```

Comment:
:thinking: Do we really need the `isFormEmbedded` field? The `formKey` seems enough to discriminate the type of form a process has.

Reply:
In general, I agree that `isFormEmbedded` is not required, but the old Taskist backend relies on it in a few places, for example, in `ProcessInternalController`. That's why I decided to keep `isFormEmbedded` to avoid refactorings in the old Tasklist backend (even if the refactoring might not appear too complex).